packages/editor/src/browser/editor-decorations-service.ts

```diff
@@ -209,7 +209,7 @@ export namespace EditorDecorationStyle {
         const index = styleSheet.insertRule('.' + selector + '{}', 0);
         const rules = styleSheet.cssRules || styleSheet.rules;
         const rule = rules.item(index);
-        if (rule.type === CSSRule.STYLE_RULE) {
+        if (rule && rule.type === CSSRule.STYLE_RULE) {
```

Comment:
What is your `tsc --version`? Is this related: https://github.com/theia-ide/theia/issues/1621?

Reply:
it's `2.7.2`, but `code-insiders` is ahead and sometimes it fails to pick `ts` from `node_modules`. that change would still be ok with `2.7.x`, I'll check that.

Reply:
Sure, and please, ignore me; I was curious. Thanks!